C++/include/sorting/utils.hpp

```diff
@@ -12,6 +12,10 @@
 
 using namespace std;
 
+//default values
+const int order = 1;
+const bool to_show_state = false;
```

Comment:
Adding these here won't solve the issue. You need to set default values for the function parameters in the definition of each sorting function that you find in various files under `source/sorting/`.

Reply:
@faheel I am on it.